console_backend/src/ntrip_tab.rs

```diff
@@ -91,7 +91,7 @@ fn build_gga(opts: &NtripOptions, last_data: &Arc<Mutex<LastData>>) -> Command {
     let (lat, lon, height, epoch) = match opts.pos_mode {
         PositionMode::Dynamic => {
             let guard = last_data.lock().unwrap();
-            (guard.lat, guard.lon, guard.alt, guard.epoch)
+            (guard.lat, guard.lon, guard.alt, None)
```

Comment:
leave this back to use System time instead of sending device gps time until this is figured out. this causes the curl to close connection. @silverjam